src/lib/style/theme.js

```diff
@@ -82,6 +82,28 @@ export const defaultTheme = {
     border: '#313131',
     info: '#434343',
   },
+  darkRebrand: {
+    statusHealthy: '#0AADA6',
+    statusWarning: '#F8F32B',
+    statusCritical: '#E84855',
+    selectedActive: '#037AFF',
+    highlight: '#1A3C75',
```

Comment:
Is it applicable to all the contexts ? (Text highlight, border for focused cards, focus on textual link, etc..)

Reply:
Most of the time, it's a combination of `selectedActive` and `highlight`.